core/controllers/admin_test.py

```diff
@@ -1047,3 +1047,27 @@ def test_clear_search_index(self):
         self.assertEqual(result_explorations, [])
         result_collections = search_services.search_collections('Welcome', 2)[0]
         self.assertEqual(result_collections, [])
+
+
+class SendDummyMailTest(test_utils.GenericTestBase):
+    """"Tests for sending test mails to admin."""
+
+    def setUp(self):
+        super(SendDummyMailTest, self).setUp()
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+
+    def test_send_dummy_mail(self):
+        self.login(self.ADMIN_EMAIL, is_super_admin=True)
+        csrf_token = self.get_new_csrf_token()
+
+        with self.swap(feconf, 'CAN_SEND_EMAILS', True):
+            generated_response = self.post_json(
+                '/sendDummyMailToAdminHandler', payload={},
+                csrf_token=csrf_token)
+            self.assertEqual(
+                generated_response['msg'], 'Success! Mail sent to admin.')
+
+        with self.swap(feconf, 'CAN_SEND_EMAILS', False):
+            generated_response = self.post_json(
```

Comment:
This is missing an assertion on the error message / the generated response. All tests should have assertions for everything. (Don't just check the status ints, check the message/response too.)

Reply:
Done.